dashboard/app/controllers/course_offerings_controller.rb

```diff
@@ -7,6 +7,7 @@ class CourseOfferingsController < ApplicationController
   def edit
     @course_offering = CourseOffering.find_by!(key: params[:key])
     render :not_found unless @course_offering
+    @self_paced_pl_course_offerings = CourseOffering.professional_learning_and_self_paced_course_offerings
```

Comment:
Since Rails methods return the last line of the method (if there's not an explicit `return` statement), I'd recommend either putting the `render :not_found` line at the very end, or add a return to the render: `return render :not_found`.

For the next PR, what do you want to happen if ` @self_paced_pl_course_offerings` returns null? Do you want the page to still load? I'm thinking yes, but want to double check.